### PR TITLE
Make "deprecated" descriptions consistent.

### PR DESCRIPTION
**Pull request by awholegunch**
Running as a Quick Build: http://builds.cask.co/browse/CDAP-DQB133-1

### CODE REVIEW

File: cdap-docs/tools/cdap-default/cdap-default-deprecated.xml [hunk 1]
```diff
@@ -176,7 +174,7 @@
     <value>0.0.0.0</value>
     <description>
       CDAP Authentication service bind address
-      (deprecated; use security.auth.server.bind.address instead)
+      (deprecated; use ${security.auth.server.bind.address} instead)
```
Comment (bdmogal) [inline cdap-docs/tools/cdap-default/cdap-default-deprecated.xml:177]: nit: semi-colon here, as opposed to colon everywhere else?

Comment (awholegunch) [reply cdap-docs/tools/cdap-default/cdap-default-deprecated.xml:177]: Fixed. I think it should be "deprecated; use ${x.y.z} instead" everywhere.

**Comment by bdmogal** [comment]
lgtm

**Comment by bdmogal** [comment]
will there be another PR to remove the metadata properties from the deprecated list in 4.0?

**Comment by awholegunch** [comment]
Yes, there will. Need to do this one and then a similar for 3.6.

**Pull request merged**